src/Arcus.Security.Core/Storage/SecretStoreBuilder.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Arcus.Security.Core;
```

Comment:
Let's move this to `Arcus.Security.Core` or `Arcus.Security.Core/Builders`

Reply:
Lets go for just `Core`, not separating them by type but by purpose.